XRTK-Core/Packages/com.xrtk.core/Services/MixedRealityToolkit.cs

```diff
@@ -488,7 +488,7 @@ private void InitializeServiceLocator()
 
             if (ActiveProfile.IsDiagnosticsSystemEnabled)
             {
-                if (!CreateAndRegisterService<IMixedRealityDiagnosticsSystem>(ActiveProfile.DiagnosticsSystemSystemType, ActiveProfile.DiagnosticsSystemProfile) || DiagnosticsSystem == null)
+                if (!CreateAndRegisterService<IMixedRealityDiagnosticsSystem>(ActiveProfile.DiagnosticsSystemSystemType, ActiveProfile.DiagnosticsSystemProfile, Instance.MixedRealityPlayspace) || DiagnosticsSystem == null)
```

Comment:
We probably don't need to pass this. Easy enough to grab from the service locator outright